core/io/xml_parser.cpp

```diff
@@ -78,13 +84,38 @@ void XMLParser::_parse_closing_xml_element() {
 	}
 
 	node_name = String::utf8(pBeginClose, (int)(P - pBeginClose));
+
+	// remove possible white spaces at the beginning
+	while(true) {
+		if(node_name.begins_with(" ")) {
+			node_name = node_name.replace_first(" ","");
+		}
+		else {
+			break;
+		}
+	}
+
+	// remove possible white spaces at the end
+	while(true) {
+		if(node_name.ends_with(" ")) {
+			node_name = node_name.reverse();
+			node_name = node_name.replace_first(" ","");
+			node_name = node_name.reverse();
+		}
+		else {
+			break;
+		}
+	}
```

Comment:
There's methods for this, you should use `strip_edges`

Reply:
Damn that's useful! Honestly, I searched in the "ustring" for a similar method but couldn't find any, I totally overlooked this one, thank you!